bundle/playwright/fixtures/pages/articles.ts

```diff
@@ -11,22 +8,13 @@ const stage = getStage();
  **/
 const articles = [
 	{
-		path: getTestUrl({
-			stage,
-			path: '/politics/2022/feb/10/keir-starmer-says-stop-the-war-coalition-gives-help-to-authoritarians-like-putin',
-		}),
```

Comment:
Remove the concept of 'stage' i.e. dev, code or prod. Hence the `getTestUrl` is no longer required